templates/resources/ignore_rule.md.tmpl

```diff
@@ -0,0 +1,15 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "xray_ignore_rule Resource - terraform-provider-xray"
+subcategory: ""
+---
+
+# xray_ignore_rule (Resource)
+
+  Provides an Xray ignore rule resource. See Xray Ignore Rules https://www.jfrog.com/confluence/display/JFROG/Ignore+Rules and REST API https://www.jfrog.com/confluence/display/JFROG/Xray+REST+API#XrayRESTAPI-IGNORERULES for more details. Notice: at least one of the 'vulnerabilities/cves/liceneses', 'component', and 'dockerlayers/artifact/build/releasebundle' should not be empty. When selecting the ignore criteria, take note of the combinations you choose. Some combinations such as omitting everything is not allowed as it will ignore all future violations (in the watch or in the system).
```

Comment:
Extra spaces at the beginning. Consider using Terraform sigil for the 'Notice:' part. (I know this is copied from API doc page but we don't have to represent it verbatim)